guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -144,7 +163,7 @@ Refer to the following table for a description of each key-value pair.
    </tbody>
 </table>
 
-#### Add a Contributor's name to a topic
+### Add a Contributor's name to a topic
 When a community member contributes an entire topic—or makes substantial improvements to an existing topic—we like to thank them by adding their name (or company name) beneath the topic title and a link to their blog or web site.
```

Comment:
"beneath the topic title with a link to their..."; and "website" not "web site"

Reply:
edited